.nycrc

```diff
@@ -1,11 +1,41 @@
 {
     "include": [
-  	  "index.js",
-	  "gpii/node_modules/*/*.js",
-	  "gpii/node_modules/*/src/*.js"
+     "**/*.js",
+     "gpii/node_modules/**/*.js"
     ],
     "exclude": [
-	  "!**/node_modules/"
```

Comment:
Why is `./node_modules` not being excluded?  You need *something* to exclude that.